src/pages/GamePage.js

```diff
@@ -10,6 +10,9 @@ import Box from "@material-ui/core/Box";
 import Snackbar from "@material-ui/core/Snackbar";
 import { Redirect } from "react-router-dom";
 
+import useSound from "use-sound";
+import boopSfx from "../assets/mySound.mp3";
```

Comment:
Can you follow the import pattern? We usually place third-party dependency module imports first in one block, then local file imports. Also, you should rename `mySound.mp3` to something more descriptive.

Finally, I think if you place the file in `assets`, it'll be served directly on the site as a static file. In which case, you should probably put it in some subdirectory, not top-level. So, `/assets/sounds/boop.mp3`, for example. And then it would be accessible from `https://setwithfriends.com/sounds/boop.mp3`, so you could remove this `import boopSfx` line and replace the `useSound(boopSfx)` line with `useSound('/sounds/boop.mp3')`.

Let me know if this sounds reasonable.